#!/usr/bin/env python
PACKAGE = "legged_controllers"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("kp_position",    double_t,    0,  "Kp_position",    5000,    0,   10000)
gen.add("kd_position",    double_t,    1,  "Kd_position",    10,    0,   10000)
gen.add("kp_big_stance",    double_t,    2,  "kp_big_stance",    150,    0,   10000)
gen.add("kp_big_swing",    double_t,    3,  "kp_big_swing",    120,    0,   10000)
gen.add("kd_big",    double_t,    4,  "kd_big",    4,    0,   2000)

gen.add("kp_small_stance",    double_t,    5,  "kp_small_stance",    100,    0,   10000)
gen.add("kp_small_swing",    double_t,    6,  "kp_small_swing",    80,    0,   10000)
gen.add("kd_small",    double_t,    7,  "kd_small",    4,    0,   10000)

gen.add("kd_feet",    double_t,    10,  "kd_feet",    4,    0,   10000)
gen.add("kp_pri",    double_t,    11,  "Kp_pri",    10000,    0,   30000)
gen.add("kd_pri",    double_t,    12,  "Kd_pri",    300,    0,   10000)
gen.add("kp_pri_walk",    double_t,    13,  "Kp_pri_walk",    1500,    0,   30000)
gen.add("kd_pri_walk",    double_t,    14,  "Kd_pri_walk",    20,    0,   10000)
exit(gen.generate(PACKAGE, "legged_controllers", "Tutorials")) 